### PR TITLE
Refactor getMembers / recent authors

### PR DESCRIPTION
**Pull request by philipperolet**
## Description
Allows querying only a subset of users via `getMembers`
Refactor recent authors to only fetch the 3 authors from memberships
## Risk
Only touches members / authors, tested locally

### CODE REVIEW

File: front/lib/api/workspace.ts [hunk 1]
```diff
@@ -104,17 +104,31 @@ export async function setInternalWorkspaceSegmentation(
...
-): Promise<UserTypeWithWorkspaces[]> {
+export async function getMembers({
+  auth,
+  role,
+  userIds,
+}: {
+  auth: Authenticator;
+  role?: RoleType;
+  userIds?: ModelId[];
+}): Promise<UserTypeWithWorkspaces[]> {
```
Comment (flvndvd) [inline front/lib/api/workspace.ts:115]: `nit` can we keep as a proper param and only bundle the optional parameters in the object?

Something like:
```suggestion
export async function getMembers(
  auth: Authenticator, {
  role,
  userIds,
}: {
  role?: RoleType;
  userIds?: ModelId[];
}): Promise<UserTypeWithWorkspaces[]> {
```

Comment (philipperolet) [reply front/lib/api/workspace.ts:115]: yeah it's better

**Comment by philipperolet** [comment]
Addressed your comment; merging; PMRW :+1:

**Comment by flvndvd** [comment]
> Addressed your comment; merging; PMRW 👍

👌

**Pull request merged**